modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetVocabularyCategoriesSelector.es.js

```diff
@@ -140,40 +140,35 @@ function AssetVocabulariesCategoriesSelector({
 			vocabularyIds: sourceItemsVocabularyIds.concat(),
 		});
 
-		const itemSelectorDialog = new ItemSelectorDialog({
+		openSelectionModal({
 			buttonAddLabel: Liferay.Language.get('done'),
-			dialogClasses: 'modal-lg',
-			eventName,
+			multiple: true,
+			onSelect: (selectedItems) => {
+				if (selectedItems) {
```

Comment:
Hey @markocikos, smallest nitpick ever, but perhaps `selectedItems.length`?

Reply:
Updated to `if (selectedItems?.length) {`